src/which.js

```diff
@@ -26,8 +29,9 @@ function checkPath(path) {
 //@ var nodeExec = which('node');
 //@ ```
 //@
-//@ Searches for `command` in the system's PATH. On Windows looks for `.exe`, `.cmd`, and `.bat` extensions.
-//@ Returns string containing the absolute path to the command.
+//@ Searches for `command` in the system's PATH. On Windows, looks for extensions in the PATHEXT
```

Comment:
*Totally optional suggestion*

Perhaps we don't need this much info in the README? The README is supposed to give a brief overview of  commands, and this might be too much detail. I think it's sufficient to say something to the effect of *"On Windows, this uses the `PATHEXT` environment variable to append the extension"* here.

I think it's better to have this comment down below in the function body.